main.js

```diff
@@ -7,6 +7,11 @@ const providers = require('./ServiceProviders/providers')
 let mainWindow
 let tray
 
+function setWindowPosition(alwaysOnTop = true) {
```

Comment:
Please move this on lib/utils.js (need to create it). This will also accept a parameter named `window` which will be the reference of the window object. Also, change the name of the function to something like `resetWindowToFloat`. Second parameter will not be required as we would have to perform the same action and the value of `windowLayer` should always be 1 as this is not about enabling it but the level in the stack of app which is applicable on macOS only.

Reply:
I was passing a widow as argument to this function previously. There is one problem with passing a window as argument to this function. Once the function completes its execution, window object gets garbage collected, so changes have done on that window object is gone, also in some cases app break because of this.